### PR TITLE
Created Allens post and Author Profile

### PR DESCRIPTION
**Pull request by Logan-Ruf**

### CODE REVIEW

File: _authors/eric.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
-title: Eric Carmichael
```
Comment (ckcollab) [inline _authors/eric.md]: I think the names of these files got goofed up? eric.md was edited

File: _posts/2020-10-26-How-finding-jobs-on-Upwork-is-like-finding-a-date.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+title:  "How finding jobs on Upwork is like finding a date?"
+date: 2020-10-26 00:00:00 +0800
+author: allen
+---
+
+
+
+Upwork, I would consider, is the best place for a Software/Web Development company based out of North Idaho to find new clients and jobs. If you are okay with a lot of rejection and getting your hopes up of finding the perfect client then it is the place for you.
+
+<!--more-->
```
Comment (ckcollab) [inline _posts/2020-10-26-How-finding-jobs-on-Upwork-is-like-finding-a-date.md]: nice, ty!

File: _posts/2020-10-26-How-finding-jobs-on-Upwork-is-like-finding-a-date.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+ Even if you have a great profile with 100% job satisfaction and great reviews, you can still find it hard to be able to get even a reply. Through my experience on Upwork, I have found a few tips that might be able to help you start to grow your profile and eventually find the perfect client.
+
+
+
+
+# Starting Out
+
+The first thing would be to grow your profile. Even if you would consider yourself an expert in your field, when hiring freelancers in the past, I would ignore profiles that had no work history or very little. This means you will have to take some work below what you would like to charge in order for your profile to grow. Even having a few jobs with 100% satisfaction can mean the difference between getting an interview or being ignored. You might have to lose some money in order to boost your profile but in the long run, you will be able to get more interviews or even have jobs come to you with extensive work history. 
+
```
Comment (ckcollab) [inline _posts/2020-10-26-How-finding-jobs-on-Upwork-is-like-finding-a-date.md]: could use some silly images in here, hmm..

File: _authors/eric.md (hunk 1 above)
Comment (sw00d) [inline _authors/eric.md]: You filled in Allen's author information on Eric.md. If we switch that up, everything else looks good!

Comment (Logan-Ruf) [reply _authors/eric.md]: Whoops! I'll fix that and merge it over. Thanks for catching that

**Pull request merged**